chain/network/src/config.rs

```diff
@@ -195,7 +198,6 @@ impl NetworkConfig {
                     .context("whitelist_nodes")?
             },
             handshake_timeout: cfg.handshake_timeout.try_into()?,
-            reconnect_delay: cfg.reconnect_delay.try_into()?,
```

Comment:
Is it intentional that `reconnect_delay` is gone? I think the corresponding field in `config_json` is now unused?

Reply:
it was unused, so I removed it from here. I guess I can deprecate it in the json config.